tests/framework/factories/class-sensei-factory.php

```diff
@@ -99,6 +99,7 @@ public function __construct() {
 		require_once dirname( __FILE__ ) . '/class-wp-unittest-factory-for-module.php';
 		require_once dirname( __FILE__ ) . '/class-wp-unittest-factory-for-message.php';
 		require_once dirname( __FILE__ ) . '/class-wp-unittest-factory-for-question-category.php';
+		require_once dirname( __FILE__ ) . '/class-sensei-unittest-factory-for-course-category.php';
```

Comment:
Why not use the same naming convention as with other factories (i.e. `class-wp-unittest`)?

Reply:
@donnapep There is a php linter that is enforcing to have the new files using the plugin name instead of the wp name (that was my initial implementation).